modules/vault-cluster/main.tf

```diff
@@ -194,15 +194,46 @@ resource "aws_s3_bucket" "vault_storage" {
   }
 }
 
+resource "aws_dynamodb_table" "vault_dynamo" {
```

Comment:
Default dynamo table with small read/write capacity.  Anything beyond (autoscaled, global tables) would need to be added or used in client code.